### DISCUSSION TITLE
Welcome to pyMOR Discussions!

### DISCUSSION THREAD

**Discussion by renefritze** [pymor/pymor#1122] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by renefritze** [comment]
Discussions are now enabled @pymor/pymor-devs

**Comment by ftalbrecht** [comment]
Hi! I'm Felix, a postdoc at the university of Münster (Germany). I am a not-too-active core developer of pyMOR and mainly interested in PDE solvers, RB methods and (spatially) localized reduced order models.

From what I understand we can use these discussions as a stackoverflow for pyMOR. In particular, turning discussions into issues seems easily possible.

So I see this as a means to start most questions and discussions, in particular if it is not yet clear whether this is an _issue with pyMOR_ or _an issue with me not understanding pyMOR_. The latter can remain here while the former can be turned into an issue along the clarification process...
